src/composeappmanager.cc

```diff
@@ -136,15 +136,38 @@ ComposeAppManager::AppsContainer ComposeAppManager::getAppsToUpdate(const Uptane
   return apps_to_update;
 }
 
+ComposeAppManager::AppsContainer ComposeAppManager::getAppsToRemove(const Uptane::Target& t) const {
```

Comment:
I'm not sure I see how this change fits in with this commit? This seems orthogonal to removal of docker-app package manager?

Reply:
It's related to removing the `LiteClient::dockerAppsChanged`.  If the docker-apps specific code is removed from that method then only invoking of `appListChanged`, the business logic of which overlaps with `ComposeAppManager::getAppsToUpdate` except determining of apps to be removed hence I added ComposeAppManager::getAppsToRemove.

Reply:
But, I think, it makes sense to make this change in two steps, at first just remove docker apps as it is and then restructure the code. BTW, it also fixes a bug with app removal if it's removed in new Target but not from the config and aklite is not restarted and running in daemon mode.